src/qt/transactiontablemodel.cpp

```diff
@@ -96,25 +96,29 @@ class TransactionTablePriv
      */
     QList<TransactionRecord> cachedWallet;
 
-    bool fQueueNotifications = false;
+    /** True when model finishes loading all wallet transaction on start */
+    bool m_loaded = false;
+    /** True when transactions are being notified, for instance when scanning */
+    int m_loading = false;
```

Comment:
(maybe a leftover from an initial version?) this can work but unless I'm misreading `m_loading` will only be set to true or false (1 or 0)
```suggestion
    bool m_loading = false;
```

Reply:
Fixed.